core/src/main/java/org/apache/iceberg/actions/BinPackStrategy.java

```diff
@@ -45,20 +44,12 @@
 abstract class BinPackStrategy implements RewriteStrategy {
 
   /**
-   * Minimum number of files that need to be in a file group to be considered
-   * for rewriting. This is considered in conjunction with {@link MIN_OUTPUT_FILES}, both
-   * conditions must pass to consider a group of files to be rewritten.
+   * The minimum number of files that need to be in a file group for it to be considered for
+   * compaction if the total size of that group is not the target size. This can also be thought of as
+   * the maximum number of non-target-size files that should remain in a file group (parititon) after rewriting.
```

Comment:
nit: `should` -> `can`?

Reply:
I couldn't say why but I think it should be "could" ... replaced

Reply:
Also typo on partition, I see now